### DISCUSSION TITLE
I just created my github profile can anyone help me with how to setup my profil;e

### DISCUSSION THREAD

**Discussion by PrasadRathnayaka** [community/community#146077] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Does any one know how to create better readme

**Comment by BhathiyaPrasad** [comment, marked as answer]
Ah Congratulations and welcome to github

1. First You can Create a Public readme file then you can show your visitors that what you are currently upto and you can place your contact information also you can create readme by simply visiting 
https://rahuldkjain.github.io/gh-profile-readme-generator/
also, you can fork my repo and use its component to make your profile readme better
https://github.com/BhathiyaPrasad/BhathiyaPrasad
2. After that pin some of your repositories that showcase your talents but remember if you have any sensitive data to add on gitignore for better secuirity

**Reply by PrasadRathnayaka** [reply]
Thankyou .. Okay

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @PrasadRathnayaka!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Profile category. I've gone ahead and moved it for you. Good luck!
